src/opticalglass/test/test_create_glass.py

```diff
@@ -8,6 +8,7 @@
 import unittest
 from opticalglass.glassfactory import create_glass
 from opticalglass import glasserror as ge
+from opticalglass import opticalmedium
```

Comment:
I have a slight preference to using:
```from opticalglass import opticalmedium as om```
and then using `om` as the module abbreviation. 
I use this convention in some **ray-optics** code and notebooks so I think its an informal convention worth reinforcing.